README.rst

```diff
@@ -99,21 +96,24 @@ Make the following changes to the ``staging.yml`` file:
 Start staging server
 ====================
 
-To start a Buildbot master on this machine run::
-
-   $ fab start:staging.yml
+Once the Docker image has built on the staging server, and the staging.yml file has been created, start the test Buildbot master from the local machine using::
 
-To update a slave on this machine, run::
+   $ fab restart:staging.yml
 
-   $ fab update:staging.yml
-
-Log in to the EC2 instance with the credentials from the ``auth`` section of the config file.
+Connect to the IP address of the EC2 instance and log in to the Buildmaster portal with the credentials from the ``auth`` section of the config file.
+Click on the ``flocker`` link to access the web form.
 
 The staging setup is missing the ability to trigger builds in response to Github pushes.
+To trigger a build, enter a branch name and click the ``Force`` button to start testing a Flocker branch.
+
+The staging master will start latent slaves on AWS EC2 automatically when builds have been triggered.
```

Comment:
The slave names do not indicate the master, but each tag has a Buildmaster tag, which does. This can be used to restrict the view just to the appropriate slaves.